libs/postgres_ffi/src/lib.rs

```diff
@@ -195,17 +198,22 @@ macro_rules! enum_pgversion {
     };
     {name = $name:ident,
      path = $p:ident,
-     typ = $t:ident,
+     $(typ = $t:ident,)?
      pgversions = [$($variant:ident : $md:ident),+ $(,)?]} => {
         pub enum $name {
-            $($variant ($crate::$md::$p::$t)),+
+            $($variant $(($crate::$md::$p::$t))?),+
         }
         impl $name {
             pub fn pg_version(&self) -> u32 {
                 enum_pgversion_dispatch!(self, $name, _ign, {
                     pgv::bindings::PG_MAJORVERSION_NUM
                 })
             }
+            pub fn pgversion(&self) -> PgMajorVersion {
```

Comment:
Can we choose better names for either of these functions? They are named too similarly

Reply:
I'm deduplicating them in favour of a single `pg_version(&self) -> PgMajorVersion`